### ISSUE TITLE
toString(), hashCode() should be string, hash

### ISSUE THREAD

**Issue by gavinking** [ceylon/ceylon.tool.converter.java2ceylon#30]
Calls to `x.toString()` and `x.hashCode()` should be automatically mapped to `x.string` and `x.hash`.

**Comment by rohitmohan96** [comment]
The converter already handles `x.toString()` to `x.string` IIRC.

**Comment by gavinking** [comment]
Nope. Didn't work for me.

**Comment by rohitmohan96** [comment]
Yes, The converter does handle `toString()` conversion. https://github.com/ceylon/ceylon.tool.converter.java2ceylon/commit/1d8265f107e6bc52ff8a00995215d0e5c46df7d3

**Comment by gavinking** [comment]
@rohitmohan96 OK, so it's true that it *sometimes* works. But it doesn't work in this case:

```java
    public void updateMetadata() {
            String albumUri = metadata.getDescription().getIconUri().toString();      
    }
```

**Comment by rohitmohan96** [comment]
Should be fixed in 1fb2c0aa47e970c3a9047c228d5e0968b4559bbd.

**Comment by gavinking** [comment]
Great, thanks man!
